runtime/include/chpl-mem-desc.h

```diff
@@ -60,6 +60,7 @@
   m(COMM_PRV_BCAST_DATA,  "comm layer private broadcast data",        false), \
   m(MEM_HEAP_SPACE,       "mem layer heap expansion space",           false), \
   m(GLOM_STRINGS_DATA,    "glom strings data",                        true ), \
+  m(STR_CAST_TO_INT,      "cast string to int",                       true), \
```

Comment:
I would be fine with using "string copy data" instead of adding a new descriptor, but I'm fine with this as well.

Reply:
OK thanks. I removed the new memory descriptor in favor of `STR_COPY_DATA`.  Look good?